integrationtests/src/test/java/io/kroxylicious/proxy/KrpcFilterIT.java

```diff
@@ -160,6 +169,46 @@ public void shouldPassThroughRecordUnchanged() throws Exception {
 
     }
 
+    @Test
+    public void shouldOfferPrometheusMetricsScrapeEndpoint() throws Exception {
+        String proxyAddress = "localhost:9192";
+
+        try (var cluster = KafkaClusterFactory.create(KafkaClusterConfig.builder().testInfo(testInfo).build())) {
+            cluster.start();
+
+            String config = """
+                    proxy:
+                      address: %s
+                    adminHttp:
+                      host: localhost
+                      port: 9193
+                      endpoints:
+                        prometheus: {}
+                    clusters:
+                      demo:
+                        bootstrap_servers: %s
+                    filters:
+                    - type: ApiVersions
+                    - type: BrokerAddress
+                    """.formatted(proxyAddress, cluster.getBootstrapServers());
+
+            var proxy = startProxy(config);
+
+            String counter_name = "test_metric_" + Math.abs(new Random().nextLong()) + "_total";
+            Metrics.counter(counter_name).increment();
+            HttpRequest request = HttpRequest.newBuilder(URI.create("http://localhost:9193/metrics")).GET().build();
```

Comment:
I wonder whether we really want to use `GET /metrics` to mean "prometheus scrapeable metrics".  _Metrics_ is a very generic term, so I wonder if we might later regret we coupling it to Prometheus in particular. I guess we could use the request `Content-Type` to server other formats if we needed. Or maybe `metrics/prometheus` would be a better path. Thoughts?

Reply:
The reason I went that way it's the prometheus prescribed default, so tooling often knows to scrape `/metrics` by default. For k8s integration that might mean one less annotation to describe the custom prometheus path. I agree it's not a nice prescription. I'll change the default to `metrics/prometheus`.

For this same reason I think we should have it configurable, users might want to expose it on `/metrics` for this reason.

Reply:
Ah, I didn't know/had forgotten that Prometheus recommended that. I'm happy to stick with `/metrics` then.